code/game/jobs/job/church.dm

```diff
@@ -165,8 +165,8 @@
 	department = DEPARTMENT_CHURCH
 	department_flag = CHURCH
 	faction = "CEV Eris"
-	total_positions = 2
-	spawn_positions = 2
+	total_positions = 4
```

Comment:
How about we set it to 3 instead, same as agrolyte?
1 preacher -> 2 acolytes -> 3 custodians and 3 agrolytes

Reply:
https://discord.com/channels/255035529085583360/265411250341543936/1147369213590245417

Reply:
> How about we set it to 3 instead, same as agrolyte? 1 preacher -> 2 acolytes -> 3 custodians and 3 agrolytes

To say here what I said on discord:

As an agrolyte main, there is barely room for two agrolytes who know what they're doing. A third without more physical room to divvy up responsibilities would be hell on earth. I'd rather take the hit to total department count.

Reply:
Adding my two credits: Technically agrolytes are also best positioned to work the bioreactor, but again if they know what they are doing it isn't so far fetched to see them solo the entire downstairs area. Extra custodians makes sense and if you wanted to I'd say move a slot from the agrolytes to custodian or acolyte roles.

Reply:
> if you wanted to I'd say move a slot from the agrolytes to custodian or acolyte roles.

I do want to swap agrolyte slot to custodian, but that would make 5 custodians which is kinda overkill in my opinion.
I think I'll leave it as it is right now and change things depending on how it goes.